src/workers/postChangelogAdded.ts

```diff
@@ -19,7 +19,12 @@ const worker: Worker = {
       return;
     }
     try {
-      await setRedisObject(REDIS_CHANGELOG_KEY, post.createdAt);
+      const timestampMs = post.createdAt / 1000; // createdAt comes as μs here
```

Comment:
@idoshamun from what I can see locally `messageToJson` returns `μs` timestamp. Could you verify this from your end?

Reply:
Yes. I'll check

Reply:
Ok I know what's the problem. It's Debezium not pubsub. When Debezium publishes a change event, it transforms all dates to microseconds. We can see that `ChangeObject` type replaces all dates with numbers. I didn't know it uses micro, I thought it's millis.
To confirm my suggestion is to try and add a new post to the `daily_updates` source, which should trigger the whole process and see what gets saved in the end.

@rebelchris if this is true (which probably it is), we have to check whether we use dates for other cdc related processing. If so, this issue occur there as well probably.

Reply:
Yep, I confirmed locally by inserting new post into the database that timestamp is in fact μs, so this should be ok.

<img width="499" alt="image" src="https://user-images.githubusercontent.com/9803078/221607239-44197b9d-b22b-4a46-9b1c-3bd1088ca612.png">

Reply:
Let's merge it then!

Reply:
@idoshamun Just to close the loop.
I checked and we don't rely on it for any other evaluations based on Debezium values.
Only community picks uses dates, but they fetch them from the DB directly.

Reply:
@rebelchris thanks for letting me know and checking that! 🙏